utils/blizzUtils.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-unused-vars */
+const { MessageEmbed, TextChannel } = require('discord.js');
+const fetch = require('node-fetch');
+const parser = require('node-html-parser');
+const blizzard = require('blizzard.js')
+require('dotenv').config()
+
+//Frostmourne ID is 3725
+//const serverID = 3725
+
+/* Returns WoW Server Status */
```

Comment:
Use JSDoc syntax for documentation purpose of function and also helps with autocomplete

https://jsdoc.app/